Population.cpp

```diff
@@ -1084,7 +1104,7 @@ int Population::transfer(Landscape* pLandscape, short landIx)
 
 // Determine whether there is a potential mate present in a patch which a potential
 // settler has reached
-bool Population::matePresent(Cell* pCell, short othersex)
+bool Population::matePresent(Species* pSpecies, Cell* pCell, short othersex)
```

Comment:
```suggestion
bool Population::matePresent(Cell* pCell, short othersex)
```
Isn't `pSpecies` already accessible as a member of `Population`?